_episodes/04-pipefilter.md

```diff
@@ -111,6 +111,14 @@ $ wc *.pdb
 > 4. `ls ethane.*`
 {: .challenge}
 
+>> ##Solution
+>> Option 3 is the solution. 
+>> Option 1 shows all files containing something, the letter `t`, something else and `ane.pdb`, this includes `octane.pdb` and `pentane.pdb`. 
+>> Option 2 shows all files contining something, `t`, any single character, `ne.` and anything else. This will give us `octane.pdb` and `pentane.pdb` but doesn't match anything which ends thane.pdb.
```

Comment:
I've updated this to use "any number and combination of characters" in place of "something".